hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -418,6 +418,43 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 		if nodePool.Status.Version == "" || isUpdatingVersion(nodePool, releaseImage.Version()) {
 			nodePool.Annotations[hyperv1.NodePoolSupportsKubevirtTopologySpreadConstraintsAnnotation] = "true"
 		}
+
+		kubevirtMachines := &capikubevirt.KubevirtMachineList{}
+
+		if err = r.Client.List(ctx, kubevirtMachines, client.InNamespace(controlPlaneNamespace)); err != nil {
+			return ctrl.Result{}, fmt.Errorf("failed to list KubeVirt Machines: %w", err)
+		}
+		allLiveMigratable := true
+		for _, kubevirtmachine := range kubevirtMachines.Items {
+			if !allLiveMigratable {
+				break
+			}
+			for _, cond := range kubevirtmachine.Status.Conditions {
+				if cond.Type == "VMLiveMigratable" && cond.Status == corev1.ConditionFalse {
```

Comment:
should be replaced with `capikubevirt.VMLiveMigratableCondition` once available.